pkg/controller/informers/core.go

```diff
@@ -295,3 +331,21 @@ func NewNamespaceInformer(client clientset.Interface, resyncPeriod time.Duration
 
 	return sharedIndexInformer
 }
+
+// NewLimitRangeInformer returns a SharedIndexInformer that lists and watches all LimitRanges
+func NewLimitRangeInformer(client clientset.Interface, resyncPeriod time.Duration) cache.SharedIndexInformer {
```

Comment:
i assume we can move everyone into their `Informer` in a single PR in a follow-on.